ci/tasks/node-build.yml

```diff
@@ -33,7 +33,7 @@ run:
       npm rebuild node-sass
       npm install
       npm run compile
-      rm -rf ./node_modules
+      rm -rf node_modules
       npm install --only=prod
```

Comment:
Shame that `--only=prod` doesn't remove any existing dev dependencies within node_modules (well the docs don't mention it does and I haven't tried it).

Reply:
Yeah, this was the original problem. I assumed that's what it did, but apparently not and @richardTowers confirmed it.